docs/content/docs/dev/table/sql/analyze.md

```diff
@@ -0,0 +1,365 @@
+- `ANALYZE TABLE` statements are triggered manually instead of automatically.
+
+
+## Run an ANALYZE TABLE statement
+
+{{< tabs "analyze table" >}}
+{{< tab "Java" >}}
+`ANALYZE TABLE` statements can be executed with the `executeSql()` method of the `TableEnvironment`. 
+
+The following examples show how to run a `ANALYZE TABLE` statement in `TableEnvironment`.
+{{< /tab >}}
+{{< tab "Scala" >}}
+`ANALYZE TABLE` statements can be executed with the `executeSql()` method of the `TableEnvironment`. 
+
+The following examples show how to run a `ANALYZE TABLE` statement in `TableEnvironment`.
+{{< /tab >}}
+{{< tab "Python" >}}
+
+`ANALYZE TABLE` statements can be executed with the `execute_sql()` method of the `TableEnvironment`.
+
+The following examples show how to run a `ANALYZE TABLE` statement in `TableEnvironment`.
+
+{{< /tab >}}
+{{< tab "SQL CLI" >}}
+
+`ANALYZE TABLE` statements can be executed in [SQL CLI]({{< ref "docs/dev/table/sqlClient" >}}).
+
+The following examples show how to run a `ANALYZE TABLE` statement in SQL CLI.
+
+{{< /tab >}}
+{{< /tabs >}}
+
+{{< tabs "a5de1760-e363-4b8d-9d6f-0bacb35b9dcf" >}}
+{{< tab "Java" >}}
+```java
+TableEnvironment tableEnv = TableEnvironment.create(...);
+
+// register a non-partition table named "Store"
+tableEnv.executeSql(
+        "CREATE TABLE Store (" +
+        " `id` BIGINT NOT NULl," +
+        " location VARCHAR(32)," +
+        " owner VARCHAR(32)" +
+        ") with (...)");
+
+// register a partition table named "Orders"
+tableEnv.executeSql(
+        "CREATE TABLE Orders (" +
+        " `id` BIGINT NOT NULl," +
+        " product VARCHAR(32)," +
+        " amount INT," +
+        " `sold_year` BIGINT", +
+        " `sold_month` BIGINT", +
+        ") PARTITIONED BY (`sold_year`, `sold_month`) "
+        ") with (...)");
+
+// Non-partition table, collect row count.
+tableEnv.executeSql("ANALYZE TABLE Store COMPUTE STATISTICS");
+
+// Non-partition table, collect row count and statistics for all columns.
+tableEnv.executeSql("ANALYZE TABLE Store COMPUTE STATISTICS FOR ALL COLUMNS");
+
+// Non-partition table, collect row count and statistics for column `location`.
+tableEnv.executeSql("ANALYZE TABLE Store COMPUTE STATISTICS FOR COLUMNS(location)");
+
+
+// Suppose table "Orders" has 4 partitions with specs:
+// Partition1 : (sold_year='2022', sold_month='8')
+// Partition2 : (sold_year='2022', sold_month='9')
+// Partition3 : (sold_year='2021', sold_month='8')
+// Partition3 : (sold_year='2021', sold_month='9')
+
+
+// Partition table, collect row count for Partition1.
+tableEnv.executeSql("ANALYZE TABLE Orders PARTITION (sold_year='2022', sold_month='8') COMPUTE STATISTICS");
+
+// Partition table, collect row count for Partition1 and Partition2.
+tableEnv.executeSql("ANALYZE TABLE Orders PARTITION (sold_year='2022', sold_month) COMPUTE STATISTICS");
+
+// Partition table, collect row count for all partitions.
+tableEnv.executeSql("ANALYZE TABLE Orders COMPUTE STATISTICS");
```

Comment:
if a table is a partition table, partition keys should be specified  ANALYZE TABLE Orders PARTITION(sold_year, sold_month) COMPUTE STATISTICS

Reply:
> if a table is a partition table, partition keys should be specified ANALYZE TABLE Orders PARTITION(sold_year, sold_month) COMPUTE STATISTICS

Done!